scripts/community_scoring.py

```diff
@@ -6,16 +6,23 @@
 the most influential communities that have the highest 
 potential of fraudulent activity. 
 '''
+import math
 import networkx as nx
-
+import pandas as pd
 
 def make_unfrozen_subgraph(G, community):
     sub_graph = G.subgraph(community)
     unfrozen_graph = nx.Graph(sub_graph)
+    for node, data in list(unfrozen_graph.nodes(data=True)):
+        if 'labels' in data and 'manifest' in data['labels']:
+            unfrozen_graph.remove_node(node)
+    for node1, node2, data in list(unfrozen_graph.edges(data=True)):
+        if data['type_'] in ['attaches_to']:
```

Comment:
I think this should be:

if ('attaches_to' or 'goes_to') in data['type_']:

Because we are trying to remove edges that were attached to the destination node.